InitGui.py

```diff
@@ -2140,22 +2140,9 @@ def getShape(keyValue=None):
 
         comboShape.blockSignals(True)
         comboShape.clear()
-        comboShape.addItem("Pie")
-        comboShape.addItem(" ") # Add separator
```

Comment:
Could you try to test if instead of a simple space " ", it doesn't crash if you use a string like the following "---------"?

Reply:
Ok, I'll try

Reply:
Crash also occurs. Also tried: 
```
if comboShape.count() > 0:
    separator_item = comboShape.model().item(comboShape.count()-1)
    if separator_item:
        separator_item.setEnabled(False)
```
and 
```
        # comboShape.blockSignals(True)
```
But also crash

Reply:
It could be even something specific of Arch Linux, maybe a bad package?